great_expectations/metrics/metric.py

```diff
@@ -118,31 +118,35 @@ def _get_metric_name(cls) -> str:
         # that a Domain exists in __bases__ should have been confirmed in MetaMetric.__new__
         raise MixinTypeError(cls.__name__, "Domain")
 
-    @classmethod
-    def _to_config(cls, model_values: dict) -> MetricConfiguration:
+    @staticmethod
+    @cache
+    def _to_config(
+        instance_class: type["Metric"], metric_value_set: frozenset
+    ) -> MetricConfiguration:
         """Returns a MetricConfiguration instance for this Metric."""
         metric_domain_kwargs = {}
         metric_value_kwargs = {}
-        for base_type in cls.__bases__:
+        metric_values = dict(metric_value_set)
```

Comment:
I don't understand this line of code. Usually you can't change a `set` to a `dict`. What is an example `metric_value_set` here?

Reply:
I see, above it's always called with a frozenset of tuples. Can you update the signature; I'll leave a comment above.